src/exchanges/crdtExchange.ts

```diff
@@ -272,14 +272,15 @@ class CrdtManager {
     this.#options = options;
   }
 
-  // TODO allow adding mutations in bulk so we can update optimistic state and execute dependent queries only once
-  addMutation(mutation: MutationOperation) {
-    this.#mutations.set(mutation.key, mutation);
+  addMutations(mutations: MutationOperation[]) {
```

Comment:
This looks good to me but we should probably hold off on merging to see if doing it this way makes persistance easier for https://github.com/Mookiies/crdt-exploration-frontend/pull/2